selfdrive/car/subaru/values.py

```diff
@@ -242,6 +242,18 @@ class CAR(Platforms):
       [SUBARU_VERSION_RESPONSE],
       whitelist_ecus=[Ecu.fwdCamera],
     ),
+    Request(
+      [StdQueries.SHORT_TESTER_PRESENT_REQUEST, SUBARU_VERSION_REQUEST],
+      [StdQueries.SHORT_TESTER_PRESENT_RESPONSE, SUBARU_VERSION_RESPONSE],
+      whitelist_ecus=[Ecu.fwdCamera],
+      logging=True,
+    ),
```

Comment:
Apparently the camera doesn't support the teser present service at all in the current diagnostic session, it gives `7f` even to the short query, so it's not even checking the subfunction yet.

https://github.com/commaai/openpilot/pull/27583#issuecomment-1474076216